daemon/commit.go

```diff
@@ -239,10 +240,13 @@ func (daemon *Daemon) Commit(name string, c *backend.ContainerCommitConfig) (str
 		if err := daemon.TagImageWithReference(id, newTag); err != nil {
 			return "", err
 		}
+		imageName = newTag.String()
 	}
 
 	attributes := map[string]string{
 		"comment": c.Comment,
+		"ImageID": id.String(),
```

Comment:
looking around I see mostly camelCase rather than title case (didn't see any title case, but could easily have missed it).
But just a nit here.